README.md

```diff
@@ -26,39 +26,57 @@ The script also enables backtraces by setting `RUST_BACKTRACE=1` env variable.
 
 ## Starting the maker and taker daemon
 
+A working example of starting both daemons with all the required command-line parameters can be found
+[here](https://github.com/itchysats/itchysats/blob/collab-settlement-test/start_all.sh#L8)
+
 The maker and taker frontend depend on the respective daemon running.
 
-At the moment the maker daemon has to be started first:
+## Starting the maker and taker frontend
 
-```bash
-cargo run --bin maker
-```
+We use a separate react projects for hosting taker and maker frontends.
+
+### Building the frontends
+
+The latest version of the built frontends will be embedded by `cargo` inside
+their respective daemons and automatically served when the daemon starts.
+Embedded frontend is served on ports `8000` and `8001` by default.
+
+This means that it is highly recommended to build the frontend _before_ the daemons.
 
-Once the maker is started you can start the taker:
+#### Taker
 
 ```bash
-cargo run --bin taker
+cd taker-frontend
+yarn install
+yarn build
 ```
 
-Upon startup the taker daemon will connect to the (hardcoded) maker and retrieve the current order.
+#### Maker
 
-Note: The sqlite databases for maker and taker are currently created in the project root.
+```bash
+cd maker-frontend
+yarn install
+yarn build
+```
 
-## Starting the maker and taker frontend
+### Developing frontend code
 
-We use a separate react projects for hosting taker and maker frontends.
+If hot-reloading of the app is required, frontend can be started in development mode.
+Development frontend is served on ports `3000` and `3001` by default.
 
-At the moment you will need a browser extension to allow CORS headers like `CORS Everywhere` ([Firefox Extension](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/)) to use the frontends.
+Note: At the moment you will need a browser extension to allow CORS headers like
+`CORS Everywhere` ([FirefoxExtension](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/)) to
+use dev frontends.
```

Comment:
I don't think that is the case, we proxy all API requests through.

Reply:
OK, seems like another stale piece of README that should've got trimmed, removed.